src/target/linux.rs

```diff
@@ -56,6 +68,68 @@ pub struct AttachOptions {
     pub kill_on_exit: bool,
 }
 
+#[derive(Debug)]
+pub struct Watchpoint {
```

Comment:
Maybe call this `HardwareBreakpoint`? You don't necessarily need to implement watchpoints on top of them.